packages/tools-iottwinmaker/src/commands/nuke.ts

```diff
@@ -24,27 +28,61 @@ export const builder: CommandBuilder<Options> = (yargs) =>
     'workspace-id': {
       type: 'string',
       require: true,
-      description: 'Specify the ID of the Workspace to delete.',
+      description: 'Specify the ID of the Workspace to delete from.',
+    },
+    'delete-workspace': {
+      type: 'boolean',
+      require: false,
```

Comment:
Can we set the default to false for these flags? That makes it obvious to developers what the default behavior is

Reply:
I'm not sure what this means, do you mean 
`default: false`?
By default (the flag is not included at all since it is not required), it is false?
Currently an example of intended behavior is: "tmdt init --region <region> --workspace-id <ws> --delete-workspace". Do you want to explicitly set "--delete-workspace true" instead? Not sure if that is any different.

Reply:
Yes I mean explicitly state it. The current behavior is correct but not clear to developers